compiler-rt/lib/ubsan/ubsan_value.cpp

```diff
@@ -120,15 +120,11 @@ FloatMax Value::getFloatValue() const {
   CHECK(getType().isFloatTy());
   if (isInlineFloat()) {
     switch (getType().getFloatBitWidth()) {
-#if 0
-      // FIXME: OpenCL / NEON 'half' type. LLVM can't lower the conversion
-      //        from '__fp16' to 'long double'.
       case 16: {
         __fp16 Value;
-        internal_memcpy(&Value, &Val, 4);
+        internal_memcpy(&Value, &Val, sizeof(Value));
```

Comment:
should we static assert that it's 4?
looks inconsistent with other branches

Reply:
it's 2, 4 is incorrect

Reply:
Of course :)
but the point that other branches use exact constants.